test/utils.dart

```diff
@@ -6,50 +6,36 @@
 
 import 'dart:io';
 
+import 'package:cli_pkg/testing.dart' as pkg;
 import 'package:path/path.dart' as p;
 import 'package:test/test.dart';
 import 'package:test_descriptor/test_descriptor.dart' as d;
 import 'package:test_process/test_process.dart';
 
+const jsExecutable = "build/sass-migrator.dart.js";
+const dartSnapshot = "build/sass_migrator.dart.snapshot";
```

Comment:
What are these used for?

Reply:
Whoops. These were leftover from before I fixed the bug in `cli_pkg`. Removed.